Overview.bs

```diff
@@ -400,46 +401,44 @@ Note: The user agent is not required to run animations on every frame. It is leg
       provide a different service level according to their policy.
 
 
-When the user agent wants to <dfn>run animators</dfn>, it <em>must</em> iterates over <a>animator
-instance list</a> as |instance|:
-
-  1. If the <a>animation requested flag</a> for the instance is <a>frame-current</a>
-        or the proxies belonging to the |instance| will not be visible within the visual viewport
-        of the current frame the user agent <em>may</em> abort all the following steps.
+When the user agent wants to <dfn>run animators</dfn> in a given |workletGlobalScope|, it
+<em>must</em> iterate over all pairs of animator name and root element as (|animatorName|,
+|rootElement|). For each such pair:
 
-        Issue: Consider giving user agents permission to skip running animator instances to throttle
-        slow animators.
+  1. Let the |definition| be the result of looking up |animatorName| on the |workletGlobalScope|'s
+        <a>animator name to animator definition map</a>.
 
-  2. Let |name| be the <a>animator name</a> of |instance|.
+      If |definition| does not exist then abort the following steps.
 
-  3. Let |workletGlobalScope| be a {{AnimationWorkletGlobalScope}} from the list of <a>worklet's
-        WorkletGlobalScopes</a> from the animation {{Worklet}}.
+  2. Let |instance| be the result of looking up (|animatorName|, |rootElement|) in the
+        |workletGlobalScope|'s <a>(animator name, root element) to instance map</a>.
 
-  4. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
-        <a>animator name to animator definition map</a>.
+      If |instance| does not exist then <a>create a new animator instance</a> for |animatorName|,
+      |rootElement|, and |workletGlobalScope|, and let |instance| be the result of that.
```

Comment:
As discussed: adding to the map is covered <dfn>create a new animator instance</dfn>.